src/train_cmd.cpp

```diff
@@ -583,6 +584,64 @@ void GetTrainSpriteSize(EngineID engine, uint &width, uint &height, int &xoffs,
 	}
 }
 
+
+/**
+ * Check if a train chain is compatible with a depot tile.
+ * @param tile Tile to check.
+ * @param t Train chain to check.
+ * @return Whether the full train chain is compatible with this tile.
```

Comment:
```suggestion
 * @return True if the full train chain is compatible with this tile, else false.
```

Reply:
I will keep the use of "@return Whether...". I am not a native English speaker, but it looks good to me. Moreover, it is used in many other comments.